curator/actions.py

```diff
@@ -1901,7 +1901,7 @@ def most_available_node(self):
         mvn_id = None
         nodes = self.client.nodes.stats()['nodes']
         for node_id in nodes:
-            name = self.client.nodes.stats()['nodes'][node_id]['name']
+            name = nodes[node_id]['name']
```

Comment:
🤦‍♂️ Thanks for this.  Can't believe I didn't think of that.

Reply:
Please add a comment to the Changelog on this in the bug fixes section.

Reply:
Done!

Reply:
I think know why it's like that now.  I probably cut and pasted what I was using in `ipython` to test (it wasn't in a loop, there).